prime-router/src/main/kotlin/cli/tests/Hl7Tests.kt

```diff
@@ -55,6 +57,54 @@ class Hl7Ingest : CoolTest() {
             passed = passed and examineLineageResults(reportId, receivers, itemCount)
         }
 
+        return passed
+    }
+}
+
+class BadHl7 : CoolTest() {
+    override val name = "badhl7"
+    override val description = "Submit badly formatted hl7 files - should get errors"
+    override val status = TestStatus.SMOKE
+    override fun run(environment: ReportStreamEnv, options: CoolTestOptions): Boolean {
+        val listBadCharacters = listOf(
```

Comment:
Explain why these are bad characters here.  The code should support UTF-8/16, so why is this bad.

Reply:
updates the test based on test cases on https://app.zenhub.com/workspaces/prime-data-hub-5ff4833beb3e08001a4cacae/issues/cdcgov/prime-reportstream/1604